packages/gatsby-source-strapi/gatsby-node.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./lib/gatsby-node');
```

Comment:
Our repo config assumes builds go to 'dist' folder, let's make that change here.